packages/terra-core-docs/src/terra-dev-site/doc/icon/example/IconThemeable.jsx

```diff
@@ -1,250 +1,251 @@
-IconInformationThemeable, 
-IconItalicI, 
-IconKeyboard, 
-IconKnurling, 
-IconLast, 
-IconLeft, 
-IconLeftPane, 
-IconLightbulb, 
-IconLink, 
-IconList, 
-IconLocationPin, 
-IconLookback, 
-IconManufacturer, 
-IconMaximize, 
-IconMediaFastForward, 
-IconMediaPause, 
-IconMediaPlay, 
-IconMediaRecord, 
-IconMediaRewind, 
-IconMediaStop, 
-IconMedicationTablet, 
-IconMedicationTabletHalf, 
-IconMedicationTabletHalfOutline, 
-IconMedicationTabletOutline, 
-IconMenu, 
-IconMerge, 
-IconMicrophone, 
-IconMicrophoneDisabled, 
-IconMinimize, 
-IconMinus, 
-IconMissedCall, 
-IconModified, 
-IconMultipleResultsCritical, 
-IconMultipleResultsNormal, 
-IconMultipleResultsNotNormal, 
-IconNavStackUp, 
-IconNext, 
-IconNoResults, 
-IconNoSignal, 
-IconNotMet, 
-IconNotification, 
-IconNotificationDisabled, 
-IconNotificationOff, 
-IconOutgoingCall, 
-IconPadlock, 
-IconPager, 
-IconPanelLeft, 
-IconPanelRight, 
-IconPaperFolded, 
-IconPaperPencil, 
-IconPartiallyMet, 
-IconPatientSearch, 
-IconPending, 
-IconPerson, 
-IconPersonConnection, 
-IconPersonDoor, 
-IconPersonHospital, 
-IconPersonnelPerson, 
-IconPharmacyReview, 
-IconPharmacyVerification, 
-IconPhone, 
-IconPhoneDown, 
-IconPill, 
-IconPinDiagonal, 
-IconPinDown, 
-IconPlusSymbol, 
-IconPlusSymbolLight, 
-IconPrevious, 
-IconPrinter, 
-IconProjects, 
-IconProtocol, 
-IconProvider, 
-IconQuestion, 
-IconQuestionOutline, 
-IconRectangleSymbol, 
-IconRectangleSymbolLight, 
-IconRecurringEvent, 
-IconRedo, 
-IconRefresh, 
-IconRefusedSlot, 
-IconReply, 
-IconReplyAll, 
-IconRequired, 
-IconRight, 
-IconRotateLeft, 
-IconRotateRight, 
-IconSave, 
-IconScratchPad, 
-IconSearch, 
-IconSecondaryProvider, 
-IconSend, 
-IconSeparate, 
-IconSettings, 
-IconSharedSchedules, 
-IconSimulation, 
-IconSortAscending, 
-IconSortDescending, 
-IconSpinner, 
-IconSquareSymbol, 
-IconSquareSymbolLight, 
-IconStartPresenting, 
-IconStopPresenting, 
-IconSwap, 
-IconTable, 
-IconTag, 
-IconTeardropSymbol, 
-IconTeardropSymbolLight, 
-IconTeardropUpSymbol, 
-IconTeardropUpSymbolLight, 
-IconThinDiamondSymbol, 
-IconThinDiamondSymbolLight, 
-IconTile, 
-IconTrash, 
-IconTreemap, 
-IconTrendingDown, 
-IconTrendingUp, 
-IconTriangleDownSymbol, 
-IconTriangleDownSymbolLight, 
-IconTriangleSymbol, 
-IconTriangleSymbolLight, 
-IconTrophy, 
-IconUnavailable, 
-IconUndo, 
-IconUnknown, 
-IconUnlock, 
-IconUnread, 
-IconUp, 
-IconUpload, 
-IconUsers, 
-IconVideoCamera, 
-IconVideoCameraDisabled, 
-IconVisualization, 
-IconVolumeSetDecrease, 
-IconVolumeSetDefault, 
-IconVolumeSetIncrease, 
-IconVolumeSetMute, 
-IconWaveform, 
-IconWifi, 
-IconWifiSlash, 
-IconXSymbol, 
-IconXSymbolLight, 
-IconZoomIn, 
-IconZoomOut, 
-} from 'terra-icon';
-
+import IconAdd from 'terra-icon/lib/icon/IconAdd';
+import IconSearch from 'terra-icon/lib/icon/IconSearch';
+import IconCommit from 'terra-icon/lib/icon/IconCommit';
+import IconCheckmark from 'terra-icon/lib/icon/IconCheckmark';
+import IconTrash from 'terra-icon/lib/icon/IconTrash';
```

Comment:
Was this intentional or a merge conflict?

Reply:
This would have been auto-built by the migration script.

Reply:
Ah okay. That should be an easy fix in a separate PR.

Reply:
Opened #4098 to update the scripts.

Reply:
Merged 4098 & updated this PR.